src/layouts/_default/baseof.html

```diff
@@ -8,7 +8,7 @@
     {{ partial "navbar" . }}
 
     <div class="container-xxl px-4 px-xxl-2">
-      <div class="d-lg-grid" id="content">
```

Comment:
We could keep the `id` in case we link to it from screenread links or something. But still use a class for styling.

Reply:
Yep I think there is no problem with this approach